### ISSUE TITLE
[Enhancement proposal] Automatically filter out hash files from assets

### ISSUE THREAD

**Issue by drzero42** [marcosnils/bin#97]
Example:
```
$ bin install github.com/docker/compose
   • Getting latest release for docker/compose

Multiple matches found, please select one:

 [1] docker-compose-Linux-x86_64
 [2] docker-compose-Linux-x86_64.sha256
 Select an option:
```

It should be a fairly easy enhancement to filter out assets ending in .sha256 or .md5 as they are very unlikely to be the assets we want to download and install.

**Comment by marcosnils** [comment]
> It should be a fairly easy enhancement to filter out assets ending in .sha256 or .md5 as they are very unlikely to be the assets we want to download and install.

Yep, I believed we talked about negative scoring attributes somehow in other issues. I can't recall if #67 or #79
